### ISSUE TITLE
Hack fix doesn't actually fix anything.

### ISSUE THREAD

**Issue by NKorotkov** [SabatinoMasala/accessoryview-demo#1]
I came across the same problem with text not being displayed during animation. I looked into your solution and it doesn't seem to work. Text is still not visible during the animation.

**Comment by SabatinoMasala** [comment]
@NKorotkov, you're absolutely right, in iOS > 9.0 the hackfix does not actually work, sorry for that!
If I find the time I'll take a look into it.
